mailpoet/views/help.html

```
@@ -1,87 +1,83 @@
-    'action': _x('Action', 'Table column heading to perform an action.'),
-    'nothingToShow': __('Nothing to show.'),
-    'preview': _x('Preview', 'Text of a link to email preview page.'),
-    'actionSchedulerStatus': __('WP Cron / Action Scheduler Status'),
-    'version': __('Version'),
-    'storage': __('Storage type'),
-    'latestActionSchedulerTrigger': _x('Next trigger run', 'Label for the next date and time of background job trigger run.'),
-    'latestActionSchedulerCompletedTrigger': _x('Last trigger run', 'Label for the last date and time of background job trigger run.'),
-    'latestActionSchedulerCompletedRun': _x('Last worker run', 'Label for the last date and time of background job worker run.'),
-  }) %>
-<% endblock %>
+<%= localize({ 'tabKnowledgeBaseTitle': __('Knowledge Base'),
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)